examples/predator_prey/movement_map.py

```diff
@@ -1,21 +1,27 @@
 
-
 def run(env, agent):
     import numpy as np
     import seaborn as sns
     import matplotlib.pyplot as plt
 
+    # Try to unwrap the environment to get the simulation manager.
+    # TODO: Should unwrap only go down to the simulation manager?
```

Comment:
Consider unwrap down to the Simulation Manager and not the AgentBased Simulation.